plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
```

Comment:
Good question. I copied these values from the current salt/files/dns/gen-zone.py.

I imagine a low zone refresh is to permit faster propagation when adding new site & server records. But, as you suggest, there may be a better way.

How can we confirm whether the slave servers accept change notifications?

I updated the zone expiration to 7 days to make the slave dns more resilient to the shadow master going offline (per a comment from Jordan, based on this happening before).